core/src/main/java/org/springframework/security/authorization/method/SecuredAuthorizationManager.java

```diff
@@ -43,10 +45,22 @@
  */
 public final class SecuredAuthorizationManager implements AuthorizationManager<MethodInvocation> {
 
-	private final AuthoritiesAuthorizationManager delegate = new AuthoritiesAuthorizationManager();
+	private AuthorizationManager<Collection<String>> delegate = new AuthoritiesAuthorizationManager();
 
 	private final Map<MethodClassKey, Set<String>> cachedAuthorities = new ConcurrentHashMap<>();
 
+	/**
+	 * Sets an {@link AuthorizationManager} that accepts a collection of authority
+	 * strings.
+	 * @param delegate the {@link AuthorizationManager} that accepts a collection of
+	 * authority strings to use
+	 * @since 6.1
+	 */
+	public void setDelegate(AuthorizationManager<Collection<String>> delegate) {
```

Comment:
I'd prefer that this method name be more descriptive. What about `setAuthoritiesAuthorizationManager`?

Reply:
Done.